app/src/main/java/com/example/parkingsystem/mvp/presenter/ParkingSizeDialogPresenter.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.parkingsystem.mvp.presenter
+
+import com.example.parkingsystem.listener.ParkingDialogListener
+import com.example.parkingsystem.mvp.contract.ParkingDialogContract
+import com.example.parkingsystem.util.Constants
+
+class ParkingSizeDialogPresenter(private val view: ParkingDialogContract.ParkingSizeDialogFragmentView) :
+    ParkingDialogContract.ParkingSizeDialogFragmentPresenter {
+
+    override fun onButtonDialogFragmentOkPressed(listener: ParkingDialogListener) {
+        val parkingSize = view.getParkingLots()
+        val parkingSizeInt = parkingSize.toInt()
+
+        if (parkingSize.isNotEmpty() && parkingSizeInt < Constants.PARKING_SIZE_MAX) {
+            view.showParkingLots(parkingSizeInt, listener)
```

Comment:
you can unify these 2 view calls into only one

Reply:
Done